### PR TITLE
[20.04] Add support for php 8

### PR DESCRIPTION
**Pull request by chris-doehring**
This PR add's support for php 8 on `20.04`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,11 @@ language: php
...
-  - nightly
+  - 8.0snapshot
 
 before_script:
   - phpenv config-rm xdebug.ini || true
   - composer update --no-interaction
 
 script:
   - vendor/bin/phpunit
-
```
Comment (svpernova09) [inline .travis.yml:13]: We should allow `8.0snapshot` to fail since 8.0 has not been released yet.

**Review by svpernova09 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (chris-doehring) [reply .travis.yml:13]: changed :)

**Review by svpernova09 — APPROVED** [review]
Thanks!

**Comment by chris-doehring** [comment]
Since php 8 will be released on Thursday, could we get a release with this change? :) @svpernova09

**Pull request merged**